doc/daikon.texinfo

```diff
@@ -260,7 +260,7 @@ There are two additional requirements.  You must have a Java 7 (or later) JVM
 The @file{tools.jar} file that comes with your JVM must also be on your
 classpath.
 
-@xref{Installing Daikon}, for situations where you should
+See bullet point two above (@ref{Installing Daikon}), for situations where you should
 follow the complete installation instructions of
 @ref{Complete installation}.  (Also, if you do not know how to add a jar file to your
```

Comment:
I left in the Simple Installation as it does work as described in the document.  Do you think it should be removed?

 

From: Michael Ernst [mailto:notifications@github.com] 
Sent: Friday, November 06, 2015 9:16 AM
To: codespecs/daikon
Cc: Mark Roberts
Subject: Re: [daikon] Move Makefile-dist into Makefile; update documentation. (#45)

 

In doc/daikon.texinfo <https://github.com/codespecs/daikon/pull/45#discussion_r44162089> :

> @@ -260,7 +260,7 @@ There are two additional requirements.  You must have a Java 7 (or later) JVM
>  The @file{tools.jar} file that comes with your JVM must also be on your
>  classpath.
>  
> -@xref{Installing Daikon}, for situations where you should
> +See bullet point two above (@ref{Installing Daikon}), for situations where you should
>  follow the complete installation instructions of
>  @ref{Complete installation}.  (Also, if you do not know how to add a jar file to your

Isn't this documentation now obsolete, along with anything else that mentions the complete installation as a contrast to other approaches?

—
Reply to this email directly or view it on GitHub <https://github.com/codespecs/daikon/pull/45/files#r44162089> .  <https://github.com/notifications/beacon/AJwyK91N1UBVmMLqnlckhRy4q6M4nL1Qks5pDNffgaJpZM4GdcSK.gif>